app/src/util/tools/formatting.ts

```diff
@@ -33,6 +33,9 @@ export const formatNumber = (number: string, decimals = 2): string => {
   if (number.length < 1) {
     return `0${decimals > 0 ? '.' + '0'.repeat(decimals) : ''}`
   }
+  if (Number(number) < 0.001 && Number(number) > 0 && decimals > 2) {
+    return '<0.001'
+  }
```

Comment:
@Frankaus This works but I don't think its the best approach. So we wanna make it so that according to number of decimal places (precision) we define it checks for that amount of decimal places.
For example :
If we have one decimal place it will evaluate to <0.1
if we have 2 decimals it will evaluate to <0.01
if we have 3 decimals it will evaluate to <0.001
if we have 4 decimal it will evaluate to <0.0001
etc....
And we should make it in one place and to do this programatically not hardcoding it...

Reply:
Definitely better approach yes. I thought of a for loop and it works on my machine. That should do the job, hope you like it